### PR TITLE
SCAN-1 Get all Bitcoin addresses that have ever been broadcast to the bitcoin network

### LINKED ISSUE #9: Setup on Linux (Ubuntu 16.04) Failing
Seems like I'm not authorized to run `./autogen.sh` from libbtc by default, adding `sudo` to `configure.sh` should fix the issue. Additionally, I ran into a compilation issue with libbloom which was resolved by running `make clean`. Running `make clean` on all dependencies before attempting to build them seems like a good idea, so I'll add that to `configure.sh`.

### PR DESCRIPTION
**Pull request by MellowYarker**
Just to get things going, this script will request addresses from [blockchain.com](https://www.blockchain.com) - after I fix #9 and get my wifi working on Ubuntu again I'll try to get this data straight from bitcoind.

This script should be called in configure.sh and should run in the background, that'll be addressed in another PR.

### CODE REVIEW

**Comment by cacheburner** [comment]
Getting all addresses from scratch would be annoying, might be better to just upload all of them as a textfile and feed it to the configure script. Could update the repo via script occasionally, or maybe just upload them then let this script download the remaining addresses.

File: src/functions.pyx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        cur.execute(start)
+        for i in new_addresses:
+            sql = "INSERT INTO usedAddresses VALUES ('{}');".format(i)
+            cur.execute(sql)
+
+        end = "COMMIT;"
+        cur.execute(end)
+        conn.close()
+    except ValueError as e:
+        # TODO: we should check if we failed the "unique" integrity constraint
```
Comment (cacheburner) [inline src/functions.pyx]: Add this then merge.

**Review by MellowYarker — COMMENTED** [review]
Worked around this by using `INSERT OR IGNORE`

**Comment by cacheburner** [comment]
The rest will be handled in SCAN-5.

**Pull request merged**